manuscript/draft.Rmd

```diff
@@ -81,53 +83,57 @@ in the future.
-* Data summary: 
+* Data summary:
   * Number of sites, species, unique observers
   * central moments of richness (mean about 51, sd about 12 spp)
   * The within-site variation was much smaller (sd about 5 spp), making the
     site-level distributions underdispersed.  
     * This underdispersion implies
-      that common count models such as binomial, Poisson, negative binomial 
+      that common count models such as binomial, Poisson, negative binomial
       won't fit well.
-    * For this reason (and because the time series methods assumed 
-      Gaussian errors), we defined model performance throughout this paper in 
+    * For this reason (and because the time series methods assumed
+      Gaussian errors), we defined model performance throughout this paper in
       terms of Gaussians (e.g. mean squared error), rather than with count
       distributions.
         * Some of the modeling approaches (SDMs and JSDMs) did use discrete
           distributions internally, however.
 * Splitting the data temporally
-  * Our data set began in `r settings$start_yr`, which was the first year that 
+  * Our data set began in `r settings$start_yr`, which was the first year that
     NDVI data was available from GIMMS
-  * All the models were trained on the data that was collected through 
+  * All the models were trained on the data that was collected through
     `r settings$last_train_year`
-  * Sites were included in the analysis if there were at least 
+  * Sites were included in the analysis if there were at least
     `r settings$min_num_yrs` years of data available during this period.
-  * The models were tested/evaluated on the data from 
-    `r settings$last_train_year + 1` `r settings$end_yr`.
+  * The models were tested/evaluated on the data from
+    `r settings$last_train_year + 1` `r settings$end_yr`.  
+
+  Bird species richness was compiled from the Breeding Bird Survey (BBS) obtained using the Data Retriever software [@pardieck2017, @morris203]. The BBS data was initially filtered to exclude all aquatic species, unidentified species, and hybrids. All data from routes that did not meet BBS acceptable criteria were excluded. Only routes within the continental U.S.A were used so as to match the climatic dataset.
+  Nineteen bioclimatic variables were derived from mean monthly temperature and precipitation from the 4km resolution PRISM dataset, allowing us to have yearly values for each variable [@prism]. This was subset to the eight variables identified by Harris [-@harris2015] to be non-correlated. For each BBS route the mean value for each variable within a 40km radius, equal to the length of a BBS route, was used to account for spatial heterogeneity. Satellite derived NDVI was from the NDIV3g dataset and was available from the time period 1981-2015 [@pinzon2014]. Elevation was from the SRTM 90m elevation dataset obtained using the R package raster [@hijmans2014, @jarvis2008].
```

Comment:
We should describe the resolution that all of the data sources are used at and that these are based on the starting coordinates for each route.